src/Console/QueueCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Laravel\Scout\Console;
+
+use Illuminate\Console\Command;
+use Laravel\Scout\Jobs\MakeRangeSearchable;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'scout:queue')]
+class QueueCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'scout:queue
+            {model : Class name of model to bulk queue}
+            {--c|chunk= : The number of records to queue in a single job (Defaults to configuration value: `scout.chunk.searchable`)}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import the given model into the search index';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $class = $this->argument('model');
+
+        $model = new $class;
+
+        $query = $model::makeAllSearchableQuery();
+
+        $min = $query->min($model->getScoutKeyName());
+        $max = $query->max($model->getScoutKeyName());
+
+        if (! $min || ! $max) {
+            $this->info('No records found for ['.$class.']');
```

Comment:
```suggestion
            $this->components->info('No records found for ['.$class.']');
```

Reply:
This would be inconsistent with the other commands in the package:

https://github.com/laravel/scout/blob/747532be6ed01ad510733065048dbaa59d4e1499/src/Console/ImportCommand.php#L51

I'll leave this up to the team if they want this 👍